src/bundle/Resources/public/scss/mixins/_type-icon.scss

```diff
@@ -0,0 +1,21 @@
+@mixin type-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: calculateRem(24px);
```

Comment:
idea - what would you think about creating mixin for setting both width and height? @dew326